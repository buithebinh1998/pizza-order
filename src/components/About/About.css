.about-article {
  display: flex;
  flex-direction: column;
  align-items: center;
}

h1 {
  color: #033a56;
  text-align: center;
}

.about-article section {
  text-align: left;
  width: 80%;
  display: flex;
  font-size: 1.2rem;
  background-color: white;
  border: none;
  align-items: center;
  margin: 1rem;
}

.about-article section div {
  padding: 0 1rem;
}

.about-article section img {
  max-width: 100%;
  height: auto;
}

@media (max-width: 992px) {
  .about-article section {
    font-size: 1rem;
    margin: 0.8rem;
  }

  .about-article section div {
    padding: 0.2rem 0.8rem;
  }

  .about-article section img {
    width: 50%;
  }
}

@media (max-width: 768px) {
  h1 {
    font-size: 1.5rem;
  }

  .about-article section {
    font-size: 0.9rem;
    margin: 0.7rem;
    width: 90%;
  }

  .about-article section div {
    padding: 0.2rem 0.7rem;
  }

  .about-article section img {
    width: 45%;
  }
}

@media (max-width: 576px) {
  h1 {
    font-size: 1rem;
  }

  .about-article section {
    font-size: 0.8rem;
    margin: 0.6rem;
  }

  .about-article section div {
    padding: 0.2rem 0.6rem;
  }

  .about-article section img {
    width: 30%;
  }
}

@media (max-width: 320px) {
  h1 {
    font-size: 0.8rem;
  }

  .about-article section {
    font-size: 0.6rem;
    margin: 0.4rem;
    width: 95%;
  }

  .about-article section div {
    padding: 0.1rem 0.4rem;
  }

  .about-article section img {
    width: 30%;
  }
}
