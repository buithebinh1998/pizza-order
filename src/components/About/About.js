import React from 'react'
import './About.css'
import about1 from './../../assets/img/about1.png'
import about2 from './../../assets/img/about2.png'
import about3 from './../../assets/img/about3.png'
const About = () => {
    return(
        <>
        <h1>ABOUT PYCOZZA</h1>
        <article className="about-article">
            <section>
                <img src={about1} alt=""/>
                <div><b>Why Pycozza is a Delivery Expert?</b><br/>
                    With more than 10.000 stores on 74 countries.<br/>
                    We bring the satisfaction to more than 1 million customers every day, more then 400 million pizza sold per year.<br/>
                    During the last 50 years, our promise "30 minute delivery" has become our standard.</div>
            </section>
            
            <section>
                <div>
                Pycozza guarantees that your order will arrive within 30 minutes or you will get a free Pizza Voucher for your next purchase!<br/>
                <b>We guarantee:</b> Voucher for 1 medium sized pizza (9 ") for the next order<br/>
                - Pycozza does not penalise late delivery staff. <br/>
                - Pycozza reserves the right to withdraw delivery without prior notice.<br/>
                - Commitments for services may be temporarily withdrawn because of poor operating conditions for delivery and will be notified at the time of receipt of the application.<br/> 
                - May not applied in abnormal or unsafe weather conditions such as heavy rain, flood, storm, tide...<br/>
                - Not applied on public holidays. 
                - Not applied for orders over 5 pizzas. 
                </div>
                <img src={about2} alt=""/>
            </section>

            <section>
                <img src={about3} alt=""/>
                <div>
                Pycozza guarantees you will be satisfied! Your pizza will alway hot and crispy when delivered to your doorstep, otherwise we will give you another!<br/>
                <b>Condition:</b> The above condition applies when there is insufficient / insufficient coating or no additive as ordered. Pycozza reserves the right to withdraw the service without prior notice. This will not apply to your order. Weather conditions such as heavy rain, floods, storms, tides...
                </div>
            </section>
        </article>
        </>
    );
}

export default About;