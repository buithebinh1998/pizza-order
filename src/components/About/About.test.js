import { testSnapshots } from "../../utils/test.util";

import About from "./About";

describe("<About / >", () => {
  testSnapshots(About, [
    {
      props: {},
      description: "default render"
    }
  ]);
});
