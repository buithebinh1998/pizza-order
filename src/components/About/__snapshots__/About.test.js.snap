// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About / > snapshots default render 1`] = `
Array [
  <h1>
    ABOUT PYCOZZA
  </h1>,
  <article
    className="about-article"
  >
    <section>
      <img
        alt=""
        src="assets"
      />
      <div>
        <b>
          Why Pycozza is a Great Pizza Delivery?
        </b>
        <br />
        With more than 15 stores on Vietnam.
        <br />
        We bring the satisfaction to more than 1000 customers every day.
        <br />
        Since the day we started our service, our promise "bring happiness to customer" has become our standard.
      </div>
    </section>
    <section>
      <div>
        Pycozza guarantees that after confirmation, your order will be delivered within 30 minutes.
        <br />
        <b>
          We guarantee:
        </b>
        <br />
        - You will get a voucher for 9'' pizza if your order arrives late.
        <br />
        - Cash-on-delivery is available, there is also ATM card payment for you.
        <br />
        - Commitments for services may be temporarily withdrawn because of poor operating conditions for delivery and will be notified at the time of receipt of the application.
        <br />
        - Not applied on public holidays.
      </div>
      <img
        alt=""
        src="assets"
      />
    </section>
    <section>
      <img
        alt=""
        src="assets"
      />
      <div>
        Pycozza guarantees you will be satisfied! Your pizza will always be kept in a warm, tasty flavor, or you will get another one for free.
        <br />
        <b>
          Condition:
        </b>
         Pycozza reserves the right to withdraw the service without prior notice. This will not apply to your order. Weather conditions such as heavy rain, floods, storms, tides...
      </div>
    </section>
  </article>,
]
`;
